vectors_deep1b/index.json

```diff
@@ -0,0 +1,24 @@
+{
+  "settings": {
+    "index.number_of_replicas": 0
```

Comment:
By not specifying the num of primary shards, the ES default of `1` will be used. Just wanted to clarify this is what you want.

Reply:
Yes, for now having 1 shard is good.